packages/amplify-graphql-function-transformer/src/__tests__/overrides/build/override.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
```

Comment:
Should this file be in git?

Reply:
this is used in unit test for transformers